relay-common/src/utils.rs

```diff
@@ -167,6 +260,29 @@ mod tests {
         let g = Glob::new("api/**/store/");
         assert!(g.is_match("api/some/stuff/here/store/"));
         assert!(g.is_match("api/some/store/"));
+
+        let g = Glob::new("/api/*/stuff/**");
+        assert!(g.is_match("/api/some/stuff/here/store/"));
+        assert!(!g.is_match("/api/some/store/"));
+    }
+
+    #[test]
+    fn test_glob_replace() {
+        let g = Glob::builder("/foo/*/bar/**")
+            .capture_star(true)
+            .capture_double_star(false)
+            .capture_question_mark(false)
+            .build();
+
+        assert_eq!(
+            g.replace_captures("/foo/some/bar/here/store", "*"),
+            "/foo/*/bar/here/store"
+        );
+        assert_eq!(g.replace_captures("/foo/testing/bar/", "*"), "/foo/*/bar/");
+        assert_eq!(
+            g.replace_captures("/foo/testing/1/", "*"),
+            "/foo/testing/1/"
+        );
```

Comment:
Should we add some more test coverage for `replace_captures`? We could do something like

```rust
for (pattern, star, double_star, question_mark, expected_result) in [
    ("**", false, true, false, "*"),
    // ...
] {
    let g = Glob::builder(pattern)
            .capture_star(star)
            .capture_double_star(double_star)
            .capture_question_mark(question_mark)
            .build();
    // test
}
```